docs/reference/feature-servers/python-feature-server.md

```diff
@@ -152,8 +152,8 @@ curl -X POST \
   }' | jq
 ```
 
-### Pushing features to the online store
-You can push data corresponding to a push source to the online store (note that timestamps need to be strings):
+### Pushing features to the online and offline stores.
+You can push data corresponding to a push source to the online store and offline stores (note that timestamps need to be strings):
```

Comment:
separate comment: I would change the curl request example below to include e.g. `to=online_and_offline` or something like that

Reply:
done.